esphome/components/remote_base/__init__.py

```diff
@@ -234,6 +234,49 @@ async def build_dumpers(config):
     return dumpers
 
 
+# Dish
+DishData, DishBinarySensor, DishTrigger, DishAction, DishDumper = declare_protocol(
+    "Dish"
+)
+Dish_SCHEMA = cv.Schema(
```

Comment:
This should be capitalized as it is a constant
```suggestion
DISH_SCHEMA = cv.Schema(
```